contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -704,7 +706,7 @@ exec_stmt_exec(PLtsql_execstate *estate, PLtsql_stmt_exec *stmt)
 				need_path_reset = true;
 			}
 	}
-	if (stmt->schema_name != '\0')
+	if (stmt->schema_name != (char *)'\0')
```

Comment:
can't we simply check stmt->schema_name != NULL?

Reply:
We can replace `strncmp(stmt->schema_name, "dbo", strlen(stmt->schema_name)) == 0)` with `strcmp(stmt->schema_name, "dbo") == 0)`

Reply:
replaced